Test "GFOIRequestInvalidSpatialFilter_foiRetrieval" fails with InvocationTargetException

Tested with http://sensorweb.demo.52north.org/52n-sos-webapp/service?service=SOS&request=GetCapabilities&AcceptVersions=2.0.0 on a local docker environment.

Selected GET and all tests.

From the test report:
```
Error in call to extension function {public org.w3c.dom.NodeList com.occamlab.te.TECore.request(org.w3c.dom.Document,java.lang.String) 
throws java.lang.Throwable}: Exception in extension function java.lang.reflect.InvocationTargetException
```

And the error log:

```
Caused by: java.lang.RuntimeException: Failed to parse resource from http://sensorweb.demo.52north.org/52n-sos-webapp/service/kvp?service=SOS&version=2.0.0&request=GetFeatureOfInterest&spatialFilter=2019-04-24T13:33:05.181Z
                        ,0.0,10.0,0.0,10.0,urn:ogc:def:crs:EPSG::4326 
 null
	at com.occamlab.te.parsers.XMLValidatingParser.parse(XMLValidatingParser.java:227)
	... 604 more
```

Request send by the test: `http://sensorweb.demo.52north.org/52n-sos-webapp/service/kvp?request=GetFeatureOfInterest&service=SOS&spatialFilter=2019-04-24T13:33:05.181Z                            ,0.0,10.0,0.0,10.0,urn:ogc:def:crs:EPSG::4326                            &version=2.0.0`

--- a/src/main/scripts/ctl/GetFeatureOfInterest.xml
+++ b/src/main/scripts/ctl/GetFeatureOfInterest.xml
@@ -1896,8 +1896,7 @@
                      <ctl:param name="version">2.0.0</ctl:param>
                      <ctl:param name="request">GetFeatureOfInterest</ctl:param>
                      <ctl:param name="spatialFilter">
-                        <xsl:value-of select="current-dateTime()" />
-                        ,0.0,10.0,0.0,10.0,urn:ogc:def:crs:EPSG::4326
+                        <xsl:value-of select="concat(current-dateTime(), ',0.0,10.0,0.0,10.0,urn:ogc:def:crs:EPSG::4326')" />
                      </ctl:param>
                      <ctl:call-function name="sosFunctions:validateExceptionReport" />
                   </ctl:request>

